.gitlab-ci.yml

```diff
@@ -72,7 +72,7 @@ variables:
 .pup_4: &pup_4
   image: 'ruby:2.1'
   variables:
-    PUPPET_VERSION: '~> 4.0'
+    PUPPET_VERSION: '~> 4.10.4'
```

Comment:
.pup_4_10 has PUPPET_VERSION = '~4.10.4'.  Why did you make this change?

Reply:
@lnemsick-simp That fixes the caching issue

Reply:
Your explanation doesn't make sense to me.  When the image was ruby:2.4, that was the caching issue. 

Again .pup_4 is supposed to be the latest 4.x version whereas .pup_4_10 is supposed to be the latest 4.10.x version

Reply:
Ah, that should be removed then since we don't need to track the latest puppet 4 any more